.github/workflows/build.yaml

```diff
@@ -32,12 +32,24 @@ jobs:
       id: go  # XXX: what is this for?
     - name: Install govendor
       run: go get -u github.com/kardianos/govendor
-    # This dominates the duration of the test. Can we get a smaller set that is
-    # enough to build embedded C parts?
-    - name: Get C dependencies
+    - name: Cache Debian dependencies
+      id: cache-deb-downloads
+      uses: actions/cache@v1
+      with:
+        path: /tmp/debian-packages/
+        key: dpkg-debian-{{ hashFiles('**/debian/control') }}
+    - name: Download Debian dependencies
+      if: steps.cache-deb-downloads.outputs.cache-hit != 'true'
+      run: |
+          sudo apt clean
+          sudo apt update
+          sudo apt build-dep -d -y ${{ github.workspace }}/src/github.com/snapcore/snapd
+          mkdir -p /tmp/debian-packages/
```

Comment:
Could we use/cache /var/cache/apt/archives directly? I'm a bit puzzled why we have an intermediate dir here

Reply:
No, because there are files we cannot tarball and this upsets the cache action. There's an issue for this upstream (each action is a github repository) and it should be eventually fixed. Once that happens we can switch to /var/.. directly.

Reply:
Specifically, the .lock file is the problem IIRC.